app/models/auction.rb

```diff
@@ -0,0 +1,9 @@
+class Auction < ApplicationRecord
+  def self.all_auctions
+    all
+  end
+
+  def self.single_auction(id)
```

Comment:
As they are, these methods look a little bit redundant with their implementation.

Also, think how they look from the caller's point of view `Auction.all_auctions` or `Auction.single_auction`. Why repeat `auction` in the method name?

Reply:
@ijpiantanida We had talked about having models be the only thing int he app that calls ActiveRecord methods, so I was creating a method on the model to wrap the ActiveRecord method that it's inheriting. It's similar for now, but I'm sure they'll diverge eventually (ex. I won't actually want to return ALL of the auctions since there will probably be pagination at some point). Do you think this is unnecessary? If I were to remove these methods, would the controller call ActiveRecord methods directly?

Reply:
One of the common tensions in Rails when doing things The Rails Way is where to put the logic. Most of the time, you end up with either overly large models or overly large controllers.

One thing to keep in mind is that these are not mutually exclusive options. In other words, you could have an abstraction that lives in between the controller and the model. However, for now, I think it would be reasonable for the controller to directly call the `ActiveRecord` methods.

Reply:
First, re. the wrappers:
I just don't see them adding a lot of value at this moment. Same as with the `image` field, I try not to add anything else other than what I need right now. Futuristic design usually doesn't pay off.
There's not much difference between calling `find` and calling something that's just a wrapper of find.

Re., models and ActiveRecord:
Rails makes this a little bit difficult because of how ActiveRecord mixes 2 concerns together: the model and persistence.
I wouldn't say that the model is the only place where ActiveRecord gets used.
It's good that you're trying to not pollute the controller with persistence concerns, but more often than not, the model is also not the right place for it. 
Model is where core business logic lives and we should try to keep it unaware of the accidental complexity of an app.
So in the example of pagination that you mentioned, I'd argue that the model is also not the right place for that logic.
Like @ericmeyer said in Slack, for now it's fine if you leave this logic unabstracted in the controller and later refactor as complexity grows.